src/cpp/session/modules/RStudioAPI.R

```diff
@@ -0,0 +1,21 @@
+#
```

Comment:
We actually declare all functions we want to expose through the rstudioapi package here: https://github.com/rstudio/rstudio/blob/master/src/cpp/r/R/Api.R

The native function you can just declare within SessionModuleContext.cpp.

After we have it working in RStudio, you should make a PR here to add it to the formal API (along with documentation): https://github.com/rstudio/rstudioapi